ide-common/src/main/kotlin/org/digma/intellij/plugin/posthog/ActivityMonitor.kt

```diff
@@ -195,6 +204,19 @@ class ActivityMonitor(private val project: Project) /*: Runnable, Disposable*/ {
         )
     }
 
+   fun registerSpanLinkClicked(source: String? = null) {
```

Comment:
I think this signature would help us more:
```suggestion
   fun registerSpanLinkClicked(panelName: String, source: String? = null) {
```

so we could know from which panel it was clicked, and source could be extra detail, for example in InsightsPanel we could provide the InsightType as the source

Reply:
added